ebpf/accesslog.proto

```diff
@@ -80,6 +80,21 @@ message AccessLogConnection {
     AccessLogConnectionTLSMode tlsMode = 4;
     // application protocol type
     AccessLogProtocolType protocol = 5;
+    // the attachment information about the connection
+    ConnectionAttachment attachment = 6;
+}
+
+message ConnectionAttachment {
+    // detect contains the environment information
+    oneof environment {
+        // detect the connection have a zTunnel environment(ambient istio)
+        ZTunnelAttachmentEnvironment zTunnel = 1;
+    }
+}
+
+message ZTunnelAttachmentEnvironment {
+    // the real destination ip address of the connection
+    string real_destination_ip = 1;
```

Comment:
Could we add an emun `by` to indicate how we resolve the real IP?

Reply:
Where do we need to use this enum?

Reply:
We don't, but we can tell which part of the logic made this. Helpful for debug only.

Reply:
I have added a new enum to describe how the rover detects and resolves the real dest IP. Is this work for you?